docs/developer/contributing/linting.asciidoc

```diff
@@ -0,0 +1,70 @@
+[[kibana-linting]]
```

Comment:
I don't see this section referenced anywhere?   Should it be under contributing/index.asciidoc?

Reply:
Whoops, good catch.